frontends/mit-open/src/pages/AboutPage/AboutPage.tsx

```diff
@@ -120,126 +111,133 @@ const AboutPage: React.FC = () => {
+              and goals, create lists of learning resources, follow topics of
+              interest, and more
+            </Typography>
           </BodySection>
           <BodySection>
-            <BodyText variant="h4" id={NON_DEGREE_LEARNING_FRAGMENT_IDENTIFIER}>
+            <Typography
+              variant="h4"
+              component="h2"
+              id={NON_DEGREE_LEARNING_FRAGMENT_IDENTIFIER}
+            >
               What is non-degree learning at MIT?
-            </BodyText>
-            <BodyText variant="body1">
-              MIT's non-degree learning programs provide specific skills,
-              knowledge, or certifications without the time commitment and
-              breadth of study required for a full degree. These programs are
-              designed to be flexible and accessible, allowing professionals,
-              students, and lifelong learners to engage with MIT's educational
-              offerings from anywhere in the world. Many of these resources are
-              free or low-cost. Whether you're looking to upskill, explore a new
-              field, or gain a deeper understanding of a subject, MIT's
-              non-degree learning resources provide a pathway for personal and
-              professional growth.
-            </BodyText>
+            </Typography>
+            <Typography variant="body1">
+              MIT's non-degree learning programs offer targeted skills,
+              knowledge, and certifications without the extensive time
+              commitment of a full degree. These programs are designed to be
+              flexible and accessible, enabling professionals, students, and
+              lifelong learners to engage with MIT's educational offerings from
+              anywhere in the world. Many resources are available for free or at
+              a low cost. Whether you want to upskill, explore a new field, or
+              deepen your understanding of a subject, MIT's non-degree learning
+              resources provide a pathway for personal and professional growth.
+            </Typography>
           </BodySection>
           <BodySection>
-            <BodyText
+            <Typography
               variant="h4"
-              id={WHAT_KINDS_OF_CONTENT_FRAGMENT_IDENTIFIER}
+              component="h2"
+              id={ACADEMIC_AND_PROFESSIONAL_CONTENT}
             >
-              What kind of content is available from MIT Open?
-            </BodyText>
-            <BodyText variant="body1">
-              MIT Open organizes non-degree educational opportunities into two
-              categories: academic and professional.
-              <br />
-              <br />
+              Academic and Professional content
+            </Typography>
+            <Typography variant="body1">
+              MIT's non-degree offerings include content developed from MIT's
+              academic and professional curriculum.
```

Comment:
from https://www.figma.com/proto/Eux3guSenAFVvNHGi1Y9Wm?node-id=7566-164686#848720013

```suggestion
              Academic and Professional curriculum.
```